bufferapp/app-shell
# Task/grow 663 target location

JIRA: [GROW-663](https://buffer.atlassian.net/browse/GROW-663)

This PR adds a new util to check whether a customer is based in India. This is for Payment Laws relating to RBI work.

We use the users local time/date settings to grab their timezone and then check if it is from India. This method isn't 100% accurate but might be sufficient WDYT?

Decided to go with just adding Indian timezones instead of functionality for all timezones, as we eventually may not need to target any countries later on.

**How to test**
Change your timezone settings in your system settings
<img width="1187" alt="Screen Shot 2022-08-16 at 3 36 44 pm" src="https://user-images.githubusercontent.com/7763710/184805767-07957c97-0b44-4c99-8526-4c708dcdf12e.png">

You can type `Intl.DateTimeFormat().resolvedOptions().timeZone` into your browser console to see which zones you're in.


**Also checked Timezones in Mixpanel Match Indian customers**
<img width="1748" alt="Screen Shot 2022-08-16 at 6 12 30 pm" src="https://user-images.githubusercontent.com/7763710/184836932-da9cc83e-3f6a-4f8f-9e5f-d595c4a4cf77.png">
